crawl-ref/source/describe.cc

```diff
@@ -1512,9 +1512,11 @@ static string _describe_ammo(const item_def &item)
                            "target, hitting an obstruction, or reaching its "
                            "maximum range.";
             break;
+#if TAG_MAJOR_VERSION == 34
```

Comment:
In addition, I think this will need `#if TAG_MAJOR_VERSION == 34`s  in the relevant enum in `item-prop-enum.h` to remove the brand entirely on the major version increment.